conda-requirements.txt

```diff
@@ -0,0 +1,41 @@
+# This file may be used to create an environment using:
```

Comment:
just use the old `pip-requirements` file for this and put the things that don't exist in anaconda into the `pip-requirements` file